activerecord/test/cases/fixtures_test.rb

```diff
@@ -1413,7 +1412,7 @@ def setup
     def teardown
       ActiveRecord::Base.configurations = @prev_configs
       ActiveRecord::Base.connection_handler = @old_handler
-      ActiveRecord::Base.connection_handlers = @old_handlers
+      clean_up_connection_handler
```

Comment:
Since we have both `connection_handler=` and `connection_handlers=` it may be worth making the new method plural to avoid confusion:

```suggestion
      clean_up_connection_handlers
```

Reply:
I did this on purpose, there will only be one connection handler in the future. 😄

Reply:
Ah, even better!

Reply:
Updated the commit message to mention this to clarify this was intentional.